### PR TITLE
inject pcp pmcd agent for remote monitoring

### PR DESCRIPTION
**Pull request by jpopelka**
Fixes https://github.com/openshiftio/openshift.io/issues/1505

### CODE REVIEW

**Comment by jpopelka** [comment]
[test]

**Comment by jpopelka** [comment]
Tested on dev

File: Dockerfile [hunk 1]
```diff
@@ -4,9 +4,17 @@ ENV LANG=en_US.UTF-8 \
     F8A_WORKER_VERSION=9e0ac13
 
 RUN useradd -d /coreapi coreapi
+
+# https://copr.fedorainfracloud.org/coprs/fche/pcp/
+COPY hack/_copr_fche_pcp.repo /etc/yum.repos.d/
+
+# https://copr.fedorainfracloud.org/coprs/jpopelka/mercator/
+# https://copr.fedorainfracloud.org/coprs/fche/pcp/
+COPY hack/_copr_jpopelka-mercator.repo hack/_copr_fche_pcp.repo /etc/yum.repos.d/
```
Comment (pkajaba) [inline Dockerfile]: you are copying it here for the second time.

**Comment by jpopelka** [comment]
Wrong rebase, PTAL.

**Pull request merged**